core/api/src/domain/phone-provider/errors.ts

```diff
@@ -10,6 +10,9 @@ export class RestrictedRegionPhoneProviderError extends PhoneProviderServiceErro
 export class UnsubscribedRecipientPhoneProviderError extends PhoneProviderServiceError {}
 export class RestrictedRecipientPhoneNumberError extends PhoneProviderServiceError {}
 export class InvalidOrApprovedVerificationError extends PhoneProviderServiceError {}
+export class MissingTypePhoneProviderError extends PhoneProviderServiceError {
+  level = ErrorLevel.Critical
```

Comment:
I'm not sure we should have Critical for this error? not sure in what scenario we would not have this, but this seems to be expected from twilio at times and not much we could about that?

Reply:
this should not be critical, we cant control external service response

Reply:
My thinking was to just flag this as critical for now but to identify what's happening more clearly so that we can look into if we should downgrade it to not alert, or if it can serve as a proxy that something is wrong with the general service.

In the last month or so it's only come up 4 time out of thousands of call ([traces](https://ui.honeycomb.io/galoy/datasets/galoy-bbw/result/jgWTYE4VR3M))

___

If we think it's better to swallow the error alert now though I can make that change